.github/workflows/build-package-docs.yaml

```diff
@@ -113,10 +113,10 @@ jobs:
       if: fromJSON(github.event.inputs.deploy)
       run: |
         echo "## Deployment details :rocket:" >> "${GITHUB_STEP_SUMMARY}"
-        echo "Publish to: ${{ github.event.inputs.deployment-environment }}" >> "${GITHUB_STEP_SUMMARY}"
-        echo "Package version: ${{ github.event.inputs.ansible-package-version }}"  >> "${GITHUB_STEP_SUMMARY}"
-        echo "Owner: ${{ github.event.inputs.repository-owner }}" >> "${GITHUB_STEP_SUMMARY}"
-        echo "Branch: ${{ github.event.inputs.repository-branch }}" >> "${GITHUB_STEP_SUMMARY}"
+        echo "Publish to: ${{ github.event.inputs.deployment-environment || 'test' }}" >> "${GITHUB_STEP_SUMMARY}"
```

Comment:
From this line down I don't think we need any default values. IIRC the need for the defaults is to allow scheduled runs so we can do nightly builds. We don't need the deploy portions of the workflow for that.

Reply:
Done